### DISCUSSION TITLE
Responsive Spline Element does not cover full screen?

### DISCUSSION THREAD

**Discussion by Gabe-Vince** [splinetool/react-spline#92] [Q&A]
Hey there

I am currently trying to integrate my spline design into a Next.js project but when I add it into my page. It only shows up at the top of the page like so:

<img width="1275" alt="Screenshot 2022-11-23 at 15 06 53" src="https://user-images.githubusercontent.com/110914409/203568494-d070c71d-fe2e-47de-835e-8ddc425b6afe.png">

I tried wrapping it in a div that occupies the full height of screen but still displays in the same position.

<img width="927" alt="Screenshot 2022-11-23 at 15 08 04" src="https://user-images.githubusercontent.com/110914409/203569176-3a3e0277-a7db-4040-b7ed-6fc09bba9011.png">

Any help would be appreciated
Mr Biskit

**Comment by marcofugaro** [comment]
Try with `<div style="height: 100vh;">`

**Reply by Gabe-Vince** [reply]
It worked. Thank you so much legend!!
